cmake/sysbuild/image_signing.cmake

```diff
@@ -74,6 +74,8 @@ function(zephyr_mcuboot_tasks)
       else()
         set(imgtool_rom_command --rom-fixed @PM_MCUBOOT_PRIMARY_ADDRESS@)
       endif()
+    else()
+      set(imgtool_rom_command --rom-fixed @PM_MCUBOOT_PRIMARY_ADDRESS@)
```

Comment:
the purpose of this is for direct-xip where you have 2 images, why is this being used when direct-xip is not used?